projects/power_distribution/src/front_uv_detector.c

```diff
@@ -0,0 +1,32 @@
+#include "can_transmit.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+
+static GpioAddress s_uv_comp_pin_address = { .port = GPIO_PORT_B, .pin = 0 };
+
+static InterruptSettings s_interrupt_settings = {
+  .type = INTERRUPT_TYPE_INTERRUPT,
+  .priority = INTERRUPT_PRIORITY_NORMAL,
+};
```

Comment:
Since this is only used in one function and never changes, I'd prefer you instead make it a local variable in `prv_register_interrupts`. That way it only takes up stack space during the function rather than all the time. (Also remove the `s_` prefix when you make it a local variable.)